app/i18n/locales/en.json

```diff
@@ -113,7 +113,6 @@
   "Check_again": "Check again",
   "Check_canned_responses": "Check on canned responses.",
   "Checked": "Checked",
-  "Choose": "Choose",
```

Comment:
Lingohub should be able to get this change and remove from other languages automatically.